### PR TITLE
feat: theme added

### PR DESCRIPTION
**Pull request by SJTGSHIVAM**
1. added multiple fixes here and there
2. added a dark theme
3. Can also add custom themes implemented functions for the same

### CODE REVIEW

File: atoms/index.css [hunk 1]
```diff
@@ -7,5 +7,6 @@
 @import "./image/index.css";
 @import "./input/index.css";
 @import "./input/index.css";
```
Comment (SJTGSHIVAM) [inline atoms/index.css]: need to remove multiple import for input element

**Review by romabulani — APPROVED** [review]
- atoms/theme/index.css is empty, you can delete it if not required
- at 2 places, there are comments, you can remove them later if not required.
Everything is looking great and good to merge

File: index.html [hunk 2]
```diff
@@ -5,6 +5,10 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./index.css" />
+    <script defer>
+      document.documentElement.style.setProperty("--tui-theme-dark", "#f00");
```
Comment (SJTGSHIVAM) [inline index.html]: need to remove this script it was there for testing purpose only

**Review by SJTGSHIVAM — COMMENTED** [review]
reviewed files from my end at once

**Review by Hnh013 — APPROVED** [review]
theme colors are well chosen fits perfectly , good to merge
custom options are very good choice of feature

**Review by varnita619 — APPROVED** [review]
Looks good to merge

**Comment by SJTGSHIVAM** [comment]
> * atoms/theme/index.css is empty, you can delete it if not required
> 
>     * at 2 places, there are comments, you can remove them later if not required.
>       Everything is looking great and good to merge

The theme feature is completed with its implementations I am yet to figure out things around it so will keep that file for a while once I understand that it is completely not needed and the whole feature has no use for the same. I will remove it. Until then I will have that skeleton in place so I don't have to make a lot of configurational changes.

**Pull request merged**